### ISSUE TITLE
Distribute colors in collaboration chart in order of segment size

### ISSUE THREAD

**Issue by pluehne** [Autodesk/hubble#131]
Currently, the collaboration chart segments are colored in clockwise order. The color palette was chosen to have sequentially contrasting colors. However, the segments usually have varying sizes, which means that less important segments might end up getting contrasting colors, while the bigger chunks are hard to distinguish.

For this reason, I’d like to distribute the colors not in clockwise order, but from the biggest to the smallest segments. In this way, the biggest chunks, which are most important, would get contrasting colors.

**Comment by larsxschneider** [comment]
In addition, it would be nice if the color of a given organization is fixed if a different time span is used.

**Comment by pluehne** [comment]
Good point! I had this in mind, and I’ll try to come up with a solution :slightly_smiling_face: (the difficulty being that the set of organizations may differ from view to view).
